Microsoft-graph-toolkit v1.1 documentation update (#6425)
microsoftgraph/microsoft-graph-docs: concepts/toolkit/components/tasks.md

}
````

#### Events
| Event | Detail | Description |
| --- | --- | --- |

> | `taskClick` | The detail contains the respective `task` object | Fired when the user interacts via `click` on a task. |

Reviewer: ```suggestion
| `taskClick` | The detail contains the respective `task` object | Fired when the user clicks or taps on a task |
```

Author: updated [taskClick](https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/commits/826c791e0854be1b0dfc11dcc6147413c40b9d8d#diff-31321da30579c0ee0299430f0959f893R97)

## Templates

The `tasks` component supports several [templates](../templates.md) that allow you to replace certain parts of the component. To specify a template, include a `<template>` element inside a component and set the `data-type` value to one of the following:

| Data type     | Data context              | Description                                                       |
| ---------     | ------------------------- | ----------------------------------------------------------------- |

> | task     | task: contains all details pertaining to task | replaces the whole default task. |

Reviewer: ```suggestion
| task     | task: a planner or to-do task object | replaces the whole default task. |
```

Author: updated [task](https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/files#diff-31321da30579c0ee0299430f0959f893R106)

> | task-details | task: contains all details pertaining to task | template replaces the details section of the task. |

Reviewer: ```suggestion
| task-details | task: a planner or to-do task object | template replaces the details section of the task. |
```

Author: updated (task-details)[https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/files#diff-31321da30579c0ee0299430f0959f893R107]
